src/SourceBuild/content/eng/Version.Details.xml

```diff
@@ -6,5 +6,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>c80f5199af3eb3ebf00ca30ec4fa3af93faf0439</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.VersionTools" Version="10.0.0-beta.25110.3">
+      <Uri>https://github.com/dotnet/arcade</Uri>
+      <Sha>c80f5199af3eb3ebf00ca30ec4fa3af93faf0439</Sha>
+    </Dependency>
```

Comment:
@mmitche If we want to keep this in sync with the other arcade dependency here (the Arcade.Sdk), we would need to update the cron job that creates these dependency updates (when doing a re-bootstrap). @dotnet/source-build can you please point us to the pipeline that handles this?